assets/js/nav.js

```diff
@@ -3,3 +3,20 @@ function toggleNavLinks() {
     link.style.display = link.style.display === "flex" ? "none" : "flex"
   })
 }
+
+function duplicate(element, {times}) {
+  for (let i = 0; i < times; i++) {
+    const clone = element.cloneNode(true)
+    clone.setAttribute("aria-hidden", "true")
+    element.parentNode.appendChild(clone)
+  }
+}
+
+document.addEventListener("DOMContentLoaded", () => {
+  const viewportWidth = document.documentElement.clientWidth || window.innerWidth;
+
+  if (viewportWidth > 768) {
+    const element = document.querySelector(".example")
+    duplicate(element, {times: 6})
+  }
+})
```

Comment:
I think we can use CSS for this. Something like

```css
  animation: slide 15s linear infinite;
@keyframes slide {
  0% {
    transform: translateX(-10%);
  }
  100% {
    transform: translateX(-100%);
  }
}
```

then we need to see if repeating the text with CSS or just using a for loop in to replicate the HTML N times.

Reply:
oh the idea is to have an animation? I thought it was just repeated. If the idea is an animation I agree that we can do it with css.

Reply:
Right, I think the idea is that with the animation we catch the attention to the next meetup section 😎